homepage.css

```diff
@@ -75,10 +75,10 @@ body{
 }
 
 #homepage-buttons button {
-    font-size: 1.8em; 
```

Comment:
refactor: using consistent spacing . it would be better to use rem, em instead of px values.